x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/workspace_panel.tsx

```diff
@@ -452,7 +453,7 @@ export const InnerWorkspacePanel = React.memo(function InnerWorkspacePanel({
     }
   }, [suggestionForDraggedField, dispatchLens]);
 
-  const IS_DARK_THEME = core.uiSettings.get('theme:darkMode');
+  const IS_DARK_THEME: boolean = useObservable(core.theme.theme$, { darkMode: false }).darkMode;
```

Comment:
I'd be nice to avoid using `useObservable` hook everywhere having to pass a **new** fallback object each time we access it. Ideally we have a hook in `packages/core/theme/core-theme-browser-internal/src/theme_service.ts` that we can easily use to access the value, similar to what we do with [`theme.useDarkMode`](https://github.com/elastic/kibana/blob/67a12674068d772df6994aa160498059801df156/src/plugins/charts/public/services/theme/theme.ts#L41-L52) react hook.

Then this would just become something like...

```suggestion
  const IS_DARK_THEME: boolean = core.theme.useDarkMode()
```

> Maybe `useDarkMode` is not the best naming but `use___`

Reply:
We can suggest this to the team that owns the theme. I think is the core team, I agree with your observation